examples/scenes/src/pico_svg.rs

```diff
@@ -0,0 +1,271 @@
+// Copyright 2024 the Vello Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! A loader for a tiny fragment of SVG
+
+use std::{num::ParseFloatError, str::FromStr};
+
+use roxmltree::{Document, Node};
+use vello::{
+    kurbo::{Affine, BezPath, Point, Size, Vec2},
+    peniko::Color,
+};
+
+pub struct PicoSvg {
+    pub items: Vec<Item>,
+    pub size: Size,
+}
+
+pub enum Item {
+    Fill(FillItem),
+    Stroke(StrokeItem),
+}
+
+pub struct StrokeItem {
+    pub width: f64,
+    pub color: Color,
+    pub path: BezPath,
+}
+
+pub struct FillItem {
+    pub color: Color,
+    pub path: BezPath,
+}
+
+struct Parser<'a> {
+    scale: f64,
+    items: &'a mut Vec<Item>,
+}
+
+impl PicoSvg {
+    pub fn load(xml_string: &str, scale: f64) -> Result<PicoSvg, Box<dyn std::error::Error>> {
+        let doc = Document::parse(xml_string)?;
+        let root = doc.root_element();
+        let mut items = Vec::new();
+        let mut parser = Parser::new(&mut items, scale);
+        let (origin, size) = if let Some(vb_attr) = root.attribute("viewBox") {
+            let vs: Vec<f64> = vb_attr
+                .split(' ')
+                .map(|s| f64::from_str(s).unwrap())
+                .collect();
+            match vs.as_slice() {
+                &[x, y, width, height] => (Point { x, y }, Size { width, height }),
+                _ => (
+                    Point::ZERO,
+                    Size {
+                        width: 300.0,
+                        height: 150.0,
+                    },
+                ),
+            }
```

Comment:
This is getting the wrong viewbox for the Forma paris-30k I have locally. It's not clear whether it's using the fallback. It might be good to have a printout for the two fallback conditions?

Reply:
This works for me on the paris-30k from the MPVG publication package, but I think it's technically incorrect when width, height, and viewBox are all defined at the same time.